plugins/ccxtExchange.go

```diff
@@ -32,6 +35,7 @@ func makeCcxtExchange(
 	apiKeys []api.ExchangeAPIKey,
 	simMode bool,
 ) (api.Exchange, error) {
+	l := logger.MakeBasicLogger()
```

Comment:
here too, don't instantiate logger here but take it in as a param to the make function

Reply:
done